rules/import.js

```diff
@@ -12,6 +12,8 @@ module.exports = {
           '**/*.spec.ts',
           '**/__tests__/**',
           '**/__mocks__/**',
+          'monitoring/**',
```

Comment:
This should also be fine, since the things inside `monitoring` is used only during the CD pipeline which uses `npm ci`, which also installs dev dependencies. Also see [this](https://github.com/ridedott/firestore-schema/pull/2691)

Reply:
Yeah, nice